source/chapters/microphones.rst

```diff
@@ -0,0 +1,317 @@
+spoken announcements, vocals, or playing a musical instrument.
+
+.. _microphones-direct-monitoring:
+
+Direct monitoring
+-----------------
+Direct monitoring with a :ref:`loopback input<microphones-loopback-input>` is 
+the recommended way to use microphones with Mixxx for most users. To use direct 
+monitoring, you need a soundcard that supports it. Most soundcards with 
+microphone inputs support direct monitoring. A soundcard 
+that also supports a loopback input is recommended for easier setup. Soundcards 
+that come built into computers typically do not support direct monitoring.
+
+Direct monitoring routes the audio from the soundcard's inputs directly to its
+outputs to avoid the latency of sending it into the computer and back out with 
+:ref:`software monitoring <microphones-software-monitoring>`. At the same time, 
+the soundcard sends the input into the computer so Mixxx can
+record and broadcast it.
+
+.. figure:: ../_static/direct-monitor-signal-path.png
+   :align: center
+   :width: 80%
+   :figwidth: 100%
+   :alt: Microphone input with direct monitoring signal path
+   :figclass: pretty-figures
+
+   Microphone input with direct monitoring signal path
+
+To setup direct monitoring, it must be
+:ref:`activated on your soundcard <microphones-activating-direct-monitoring-on-soundcard>`.
+If you are not recording or broadcasting the microphone with the music, no
+further setup is needed. To record and/or broadcast with the microphone
+signal mixed into the music, set up a either a 
+:ref:`loopback input<microphones-loopback-input>` or
+:ref:`latency compensation <microphones-latency-compensation>`.
+
+Because the microphones are mixed with the music by the soundcard rather 
+than Mixxx, adjust the microphone volume with the input gain knob on 
+your soundcard. If you adjust the microphone gain knob in Mixxx, the volume 
+of the microphone relative to the music will be different in your recorded 
+and broadcasted mixes compared to what you hear from your soundcard's output.
+
+.. _microphones-loopback-input:
+
+Loopback Input
+^^^^^^^^^^^^^^
+Some soundcards that support direct monitoring have a feature called
+"loopback". Using direct monitoring with a soundcard that supports loopback is 
+the recommended way to use microphones with Mixxx. It is the easiest to 
+setup without the latency issue of
+:ref:`software monitoring <microphones-software-monitoring>` and cheapest 
+to set up with good sound quality.
+
+Using direct monitoring, the microphone input is mixed with the music output 
+from Mixxx by the soundcard. This avoids the latency of sending the
+microphone signal into the computer and back out. The mixed signal is sent to
+the speakers connected to the main output. Using the loopback feature, that
+mixed signal is sent into the computer instead of the microphone signal alone,
+typically using input channels 1-2.
+
+.. figure:: ../_static/direct-monitor-loopback-signal-path.png
```

Comment:
I would like to make a diagram similar to the block diagram of the Roland Rubix 24 on page 26 of [its manual](https://static.roland.com/assets/media/pdf/Rubix_reference_eng01_W.pdf) but excluding the details in the analog domain. The parts between the A/D and D/A converters in that diagram are what matter for explaining this.